cmd/pdatagen/internal/base_fields.go

```diff
@@ -55,6 +55,25 @@ const accessorsMessageTestTemplate = `func Test${structName}_${fieldName}(t *tes
 	assert.EqualValues(t, generateTest${returnType}(), ms.${fieldName}())
 }`
 
+const accessorsMessageValueTemplate = `// ${fieldName} returns the ${lowerFieldName} associated with this ${structName}.
+// If no ${lowerFieldName} available, it creates an empty message and associates it with this ${structName}.
+//
+//  Empty initialized ${structName} will return "nil" ${returnType}.
+//
+// Important: This causes a runtime error if IsNil() returns "true".
```

Comment:
This comments no longer apply, since it is not a pointer anymore.

Reply:
It still applies because this is the getter for a non double pointer message, but the parent struct that embeds this message can still be double pointer.